CONTRIBUTING.md

```diff
@@ -49,3 +49,6 @@ You don't have to update existing files' license comments.
 ## Documentation
 
 See the [documents](https://ebiten.org/documents/implementation.html) about internal implementation.
+
+Before submitting a documentation PR, read through the [README in the docs
+repo](https://github.com/ebiten/ebiten.org/blob/main/README.md).
```

Comment:
This section is about 'documentations about internal details', rather than 'how to fix documentations'.

So, can we

* rename the existing `Documentation` section to `Implementation details`
* add a new section `Fixing documentation` for your new paragraph

?

Reply:
And, adding a paragraph like `Ebiten's documentations are managed at github.com/ebiten/ebiten.org` to the head would be nice.

Reply:
Wilco. I can do that.

Reply:
OK, check those changes and see if you want things worded differently.

Reply:
Perfect!